evaluator/evaluator_test.go

```diff
@@ -105,3 +105,29 @@ func TestIfElseExpression(t *testing.T) {
 		}
 	}
 }
+
+type ReturnTest struct {
+	input    string
+	expected interface{}
+}
+
+func TestReturnStatement(t *testing.T) {
+	tests := []ReturnTest{
+		{input: "return 5;", expected: 5},
+		{input: "return -10;", expected: -10},
+		{input: "return 1-2;", expected: 1 - 2},
+		{input: "return 1-2+3;", expected: 1 - 2 + 3},
+		{input: "return true;", expected: true},
+		{input: "return false;", expected: false},
+	}
```

Comment:
```suggestion
	tests := []ReturnTest{
		{input: "return 5;", expected: 5},
		{input: "return -10;", expected: -10},
		{input: "return 1-2;", expected: 1 - 2},
		{input: "return 1-2+3;", expected: 1 - 2 + 3},
		{input: "return true;", expected: true},
		{input: "return false;", expected: false},
		{input: "return null;", expected: nil},
	}
```
don't forget to handle `tt.expected` being null